torch/_torch_docs.py

```diff
@@ -152,6 +152,28 @@ def merge_dicts(*dicts):
     tensor([ 1.2294,  2.2004,  1.3690,  1.7298])
 """.format(**common_args))
 
+add_docstr(torch.acosh,
+          r"""
+acosh(input, out=None) -> Tensor
+
+Returns a new tensor with arccosh (inverse hyperbolic cosine) of the elements of :attr:`input`.
+
+.. math::
+    \text{out}_{i} = \cosh^{-1}(\text{input}_{i})
+""" + r"""
+Args:
+    {input}
+    {out}
+
+Example::
+
+    >>> a = torch.randn(4).uniform_(1, 2)
```

Comment:
I had to explicitly set the range (since domain for `acosh` and `asinh` is `[1, inf)`) using `uniform_` else it will give `nan` for values out of the domain. Is this format fine?

Reply:
It would be nice if the range was included in the documentation. For example, a second sentence like: "All values in the input tensor must be greater than or equal to one." 

Also, the torch.cos documentation describes {input} and {out}, we should copy their language.